src/utils/uid.js

```diff
@@ -8,7 +8,7 @@ import generate from 'uid'
  */
 
 function uid() {
-  return generate(4)
+  return String(N++)
```

Comment:
We're running into issues with this, where the document and the first text node both have the id `"0"`. Can this be some sort of race condition?